src/config.ts

```diff
@@ -29,3 +29,5 @@ export const SEARCH_DELAY = 1000;
 export const DOWNLOADABLE_APP_FILENAME = 'index.html';
 
 export const SYNC_TOASTS_CONFIG = { timeout: 2000, pos: 'bottom_right' } as ToastItem;
```

Comment:
@Valexr https://github.com/basf/bscience-gui/blob/d7324b2c7d6c0b8c2180e60098ff99494bbca4cb/src/config.ts#L31 does not apply, right?